### ISSUE TITLE
Doesn't work if no project is open

### ISSUE THREAD

**Issue by Insiderser** [scana/ok-gradle#20]
IDE: Android Studio 3.6 RC 1
OS: Windows 10 1909
`OK, Gradle!` version: 0.3.0

#### Steps to reproduce
1. Launch Android Studio, but don't open any project.
2. Using `Ctrl + Shift + A` shortcut, launch `OK, Gradle!` action.
3. `OK, Gradle!` window doesn't open. In IDE Fatal Errors window, see the following stack trace:
```
kotlin.TypeCastException: null cannot be cast to non-null type com.intellij.openapi.project.Project
	at me.scana.okgradle.OkGradleAction.actionPerformed(OkGradleAction.kt:29)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:265)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:282)
	at com.intellij.ide.actions.GotoActionAction.lambda$performAction$7(GotoActionAction.java:335)
	at com.intellij.openapi.application.TransactionGuardImpl.runSyncTransaction(TransactionGuardImpl.java:82)
	at com.intellij.openapi.application.TransactionGuardImpl.lambda$submitTransaction$1(TransactionGuardImpl.java:106)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:115)
	at com.intellij.openapi.application.TransactionGuardImpl.lambda$submitTransactionLater$4(TransactionGuardImpl.java:268)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:433)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

#### Expected behavior
`OK, Gradle!` window gets opened and it's possible to search for dependencies.

P.S. It looks like the plugin requires a project to be open. In that case, it works as usual.

**Comment by scana** [comment]
Thank you for your report @Insiderser :)
I will add  a `headless` mode, in which one will be only allowed to copy statements :)

**Comment by scana** [comment]
Thank you once again @Insiderser - this has been fixed in v0.3.1, should be available in JetBrains repository soon!
